### PR TITLE
Adding UCL examples & editorial updates

### PR DESCRIPTION
**Pull request by QiufangMa**
Add examples showing how group-based ACLs are used. The current examples haven't consider the schedule mechanism yet, which needs to be added in following commits. But I'd like to let my co-authors to review this part firstly.

### CODE REVIEW

**Review by danielkinguk — APPROVED** [review]
The proposed changes fix a few NITs, I'd like to do a full review of the document text to squash some additional typos.

File: draft-ma-opsawg-ucl-acl.md [hunk 1]
```diff
@@ -580,9 +592,74 @@ CoA-Request CoA-ACK CoA-NACK #        Attribute
...
 --- back
 
+# Example Usage
+
+## Configuring the Controller Using Group based ACL {#controller-ucl}
+
+   Assume the organization would like to restrict the access of R&D
+   employees that bring personally owned devices into the workplace.
+
+   Access requirements are following:
```
Comment (boucadair) [inline draft-ma-opsawg-ucl-acl.md:602]:
```suggestion
   The access requirements are as follows:
```

File: draft-ma-opsawg-ucl-acl.md [hunk 2]
```diff
@@ -580,9 +592,74 @@ CoA-Request CoA-ACK CoA-NACK #        Attribute
 
 --- back
 
+# Example Usage
+
+## Configuring the Controller Using Group based ACL {#controller-ucl}
+
+   Assume the organization would like to restrict the access of R&D
```
Comment (boucadair) [inline draft-ma-opsawg-ucl-acl.md:599]:
```suggestion
   Let's consider an organization that would like to restrict the access of R&D
```

File: draft-ma-opsawg-ucl-acl.md [hunk 3]
```diff
@@ -580,9 +592,74 @@ CoA-Request CoA-ACK CoA-NACK #        Attribute
...
+   employees that bring personally owned devices into the workplace.
```
Comment (boucadair) [inline draft-ma-opsawg-ucl-acl.md:600]:
```suggestion
   employees that bring personally owned devices (BYOD) into the workplace.
```

File: draft-ma-opsawg-ucl-acl.md [hunk 4]
```diff
@@ -580,9 +592,74 @@ CoA-Request CoA-ACK CoA-NACK #        Attribute
...
+
+   Access requirements are following:
+
+   * Permit traffic from R&D BYOD employees, destined to R&D employees.
```
Comment (boucadair) [inline draft-ma-opsawg-ucl-acl.md:604]:
```suggestion
   * Permit traffic from R&D BYOD of employees, destined to R&D employees' devices.
```

File: draft-ma-opsawg-ucl-acl.md [hunk 5]
```diff
@@ -580,9 +592,74 @@ CoA-Request CoA-ACK CoA-NACK #        Attribute
...
+
+   * Deny traffic from R&D BYOD employees, destined to finance servers
```
Comment (boucadair) [inline draft-ma-opsawg-ucl-acl.md:606]:
```suggestion
   * Deny traffic from R&D BYOD of employees, destined to finance servers
```

File: draft-ma-opsawg-ucl-acl.md [hunk 6]
```diff
@@ -580,9 +592,74 @@ CoA-Request CoA-ACK CoA-NACK #        Attribute
...
+     located in the enterprise DC network.
+
+   The following example illustrates configuring the SDN controller
```
Comment (boucadair) [inline draft-ma-opsawg-ucl-acl.md:609]:
```suggestion
   The following example illustrates the configuration of the SDN controller
```

File: draft-ma-opsawg-ucl-acl.md [hunk 7]
```diff
@@ -580,9 +592,74 @@ CoA-Request CoA-ACK CoA-NACK #        Attribute
...
+   using the group-based ACL:
+
+~~~~
+{::include ./examples/controller-ucl.xml}
+~~~~
+
+## Configuring the PEP Using Group based ACL {#PEP-ucl}
+
+   This section illustrates an example configuring the PEP device using
```
Comment (boucadair) [inline draft-ma-opsawg-ucl-acl.md:618]:
```suggestion
   This section illustrates an example to configure a PEP  using
```

File: draft-ma-opsawg-ucl-acl.md [hunk 8]
```diff
@@ -580,9 +592,74 @@ CoA-Request CoA-ACK CoA-NACK #        Attribute
...
+   shows ACL configurations delivered from the controller to the PEP. This
+   example is consistent with the example presented in {{controller-ucl}}.
+
+~~~~
+{::include ./examples/PEP-ucl.xml}
+~~~~
+
+## Configuring the PEP Using Address based ACL {#PEP-acl}
+
+   The section illustrates an example configuring the PEP device using
```
Comment (boucadair) [inline draft-ma-opsawg-ucl-acl.md:639]:
```suggestion
   The section illustrates an example of configuring a PEP using
```

**Pull request merged**